### ISSUE TITLE
TypeException when sending mail on Craft 3.4.28.1

### ISSUE THREAD

**Issue by stenvdb** [putyourlightson/craft-sendgrid#8]
I'm getting the following error below when dumping `Craft::dd($e);` on line https://github.com/putyourlightson/craft-sendgrid/blob/v1/src/mail/SendgridTransport.php#L58 when sending the Craft test email from the control panel.
```
'Number of elements in \"$ccEmails\" can not be less than 1.'
```

I didn't really got any further than this when debugging, but I can't send mail anymore using this plugin. When I'm switching to SendGrid's SMTP relay the same API key is working, so that cannot be the issue. 

This happens on Craft 3.4.28.1. I can confirm the exact same setup does work in Craft 3.4.25. Sendgrid plugin is latest version. I have a feeling the error is caused by a change in this issue (released in Craft 3.4.28): https://github.com/craftcms/cms/issues/6259

Could you have a look?

**Comment by bencroker** [comment]
Thanks, fixed in https://github.com/putyourlightson/craft-sendgrid/commit/f89c452632c61c32d8f5c9d7990610513b8d1092 and released as 1.2.1.

**Comment by stenvdb** [comment]
That was fast ⚡ Tested it, it works!
